### ISSUE TITLE
library crashes when writing properties for object

### ISSUE THREAD

**Issue by rmelian** [asyncapi/asyncapi-react#69]
**Description**

When writing a document, specifically when adding new properties to an object, the library throws an error 

**Expected result**

ignore properties when invalid

**Actual result**

![image](https://user-images.githubusercontent.com/4565267/64900314-45555180-d65e-11e9-8165-a41ea867343a.png)

**Steps to reproduce**

```yaml
asyncapi: '2.0.0'
id: 'urn:myapi'
info:
  title: My API
  version: '1.0.0'
channels:
  mychannel:
    publish:
      message:
        $ref: '#/components/messages/testMessage'

components:
  messages:
    testMessage:
      payload:
        $ref: '#/components/schemas/testSchema'
  schemas:
    testSchema:
      type: object
      properties:
        name:
          type: string
    order:
      type: object
      properties:
        name:
          type: string
    activity:
      type: object
      properties:
        name:
```

**Troubleshooting**

This is the validatedSchema returned from AsyncApi.tsx

![image](https://user-images.githubusercontent.com/4565267/64900433-2905e480-d65f-11e9-9ff5-28cc43f800bf.png)

**Comment by rmelian** [comment]
I have a fix for this too, I'll send it after #67 gets fixed

**Comment by derberg** [comment]
Fixed with https://github.com/asyncapi/asyncapi-react/pull/68
This week new version of the package will be released
